coa-helm/values.yaml

```diff
@@ -37,10 +37,19 @@ config:
   cbasUI:
     conf_file: config.json
     conf_dir: /usr/share/nginx/html
+  wds:
+    conf_dir: /etc/conf
+    conf_file: wds.yaml
+  postgres:
+    startup_dir: /docker-entrypoint-initdb.d
+    helper_dir: /wds-helpers
+    startup_scripts:
+      # Numbered because scripts in initdb.d are run in alphabetical order:
+      create_dbs: 01_create_dbs.sh
+      seed_wds: 02_seed_wds.sql
 
 images:
-  mysql: mysql:latest
-  cromwell: broadinstitute/cromwell:79
+  cromwell: broadinstitute/cromwell:84-3a19be6-SNAP

```

Comment:
Do we need to update the [Cromwell update pusher](https://github.com/broadinstitute/cromwell/actions/workflows/chart_update_on_merge.yml) to target the CoA as well as GCP versions of the chart?

Reply:
Maybe we should... I admit that I haven't fully internalized all of the automated actions that organizes this repo

Reply:
I think we should, assuming we shouldn't instead refactor this to a single image name in the library chart.